include/SSVOpenHexagon/Global/Assets.hpp

```diff
@@ -71,6 +71,16 @@ class HGAssets
     [[nodiscard]] bool loadPackData(const ssvufs::Path& packPath);
     [[nodiscard]] bool loadPackInfo(const PackData& packData);
 
+    void setLevelFavoriteFlag(const std::string& mAssetId, const bool flag)
+    {
+        getEditLevelData(mAssetId).favorite = flag;
+    }
+
+    bool checkLevelIDPurity(std::string& mAssetId)
```

Comment:
```suggestion
    [[nodiscard]] bool checkLevelIDPurity(const std::string& mAssetId)
```

The name is also a bit confusing...